### PR TITLE
Update the AMI list with missing regions

### PR DESCRIPTION
**Pull request by mpas**
Update the list with usable ECS optimized ami's

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -53,25 +53,27 @@ variable "subnet_ids" {
...
+    eu-west-3      = "ami-ca75c4b7" # EU (Paris)
+    eu-central-1   = "ami-10e6c8fb" # EU (Frankfurt)
+    ap-northeast-1 = "ami-f3f8098c" # Asia Pacific (Tokyo)
+    ap-northeast-2 = "ami-7c69c112" # Asia Pacific (Seoul)
+    ap-southeast-1 = "ami-b75a6acb" # Asia Pacific (Singapore)
+    ap-southeast-2 = "ami-bc04d5de" # Asia Pacific (Sydney)
+    ca-central-1   = "ami-da6cecbe" # Canada (Central)
+    ap-south-1     = "ami-c7072aa8" # Asia Pacific (Mumbai)
+    sa-east-1      = "ami-a1e2becd" # South America (São Paulo)
+    us-gov-west-1  = "ami-a1e2becd" # GovCloud
```
Comment (JeroenKnoops) [inline variables.tf]: This is not correct.... this should be: `ami-03920462`

**Review by JeroenKnoops — CHANGES_REQUESTED** [review]
This was already merged..

File: variables.tf (hunk 1 above)
Comment (mpas) [reply variables.tf]: The review process is working! :) Thanks!

**Comment by JeroenKnoops** [comment]
Closes #8

**Pull request merged**